test/trans.render.object.spec.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { mount } from 'enzyme';
```

Comment:
Should probably remote the unused import `mount` from enzyme.

Reply:
Yeah, remove if it is not unused :partying_face:, I would also say we should not import `screen`, if it is only used in comments :raised_hands: .

Reply:
I probably forgot to remove it. I just removed it thanks